Packs/HarfangLabEDR/ReleaseNotes/1_0_2.md

```diff
@@ -0,0 +1,71 @@
+
+#### Integrations
+##### HarfangLab EDR
+
+- Fixed bug triggered when an agent (Linux) has no associated domainname
```

Comment:
It's look like this was entered here by mistake
did you change this integration ?
i see change only in the `Hurukai` integration